contrib/QtCreator/MEGAclc/MEGAclc.pro

```diff
@@ -1,58 +1,23 @@
-TEMPLATE = app
 CONFIG += console c++11
```

Comment:
The config `c++11` is already added in the `megachat.pri`

Reply:
Removed